libticalcs/trunk/tests/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 cmake_minimum_required(VERSION 3.12)
 
 project(libticalcs2-tests
-        LANGUAGES   C CXX)
+        LANGUAGES   CXX)
 
-add_executable(torture_ticalcs torture_ticalcs.c)
+add_executable(torture_ticalcs torture_ticalcs.cc)
```

Comment:
Good, but you need to modify the Makefile.am as well :)

Reply:
Indeed. Done.